students/kenneth_murray/Lesson8/Activity/NoSQL/learnnosql.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""
+
+Integrated example for nosql databases
+Lesson 8
+Ken Murray
+
+
+"""
+
+import learn_data
+import mongodb_script
+import redis_script
+import neo4j_script
+import simpler_script
+import utilities
+
+
+def showoff_databases():
+    """
+    Here we illustrate basic interaction with nosql databases
+    """
+
+    log = utilities.configure_logger('default', '..//logs//nosql_dev.log')
```

Comment:
I would suggest using something like absolute path when specifying a log location so that no matter where the program is run it will put it in the right spot.  So basically defining the location from the root of the drive.